### DISCUSSION TITLE
[Font-Optimization] Add support for Adobe fonts (Typekit)

### DISCUSSION THREAD

**Discussion by andrewgadziksonos** [vercel/next.js#18167] [Ideas]
Hello!

My company uses Adobe Fonts as their web font platform and we are looking to add support into NextJS to optimize those fonts.

I've recently created a PR for adding font-optimization support for adobe fonts / typekit.  Changes can be reviewed here, https://github.com/vercel/next.js/pull/18146

I wasn't sure what the appropriate process was for adding a new feature, so I hope this is the right place 🤞   If this isn't the right place, please point me in the right direction!

Thanks,

Andrew

**Comment by oelbaga** [comment]
Was this ever added to latest Next. We also use Adobe fonts typekit. I can't believe this isn't supported by now.
